.github/CODEOWNERS

```diff
@@ -159,10 +158,6 @@
 /plugins/nsatz/         @thery
 # Secondary maintainer @ppedrot
 
-/plugins/omega/         @letouzey
-
-/plugins/romega/        @letouzey
```

Comment:
Can't you keep this and use `@nobody`?

Reply:
`@nobody` is a real person though so you probably meant `@ghost`.

Reply:
The other day I discovered you can cite "teams", eg `@coq/pushers`, so maybe this is what we should do...